crates/uv/tests/it/pip_compile.rs

```diff
@@ -413,6 +413,250 @@ fn compile_constraint_extra() -> Result<()> {
     Ok(())
 }
 
+/// This is a regression test for a case where `uv pip compile --universal`
+/// would include dependencies with marker expressions that always evaluate
+/// to false (for example, `python_version < '0'`).
+///
+/// See: <https://github.com/astral-sh/uv/issues/8676>
```

Comment:
This test is slow (6s - 7s), can we replace it by a faster, more minimal example? This problem should model well in packse.